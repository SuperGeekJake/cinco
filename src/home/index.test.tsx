import { vi } from "vitest";
import { render } from "@solidjs/testing-library";

import { Home } from "./index";

vi.mock("@solidjs/router", () => ({
  A: (props: any) => <a {...props} />,
}));

describe("Home component", () => {
  test("matches the snapshot", () => {
    const { container } = render(() => <Home />);
    expect(container.innerHTML).toMatchSnapshot();
  });
});
