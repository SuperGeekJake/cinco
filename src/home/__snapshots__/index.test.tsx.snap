// Vitest Snapshot v1

exports[`Home component > matches the snapshot 1`] = `"<main class=\\"flex flex-col h-screen p-10\\"><h1>Cinco</h1><a class=\\"block w-full bg-light text-dark\\" href=\\"/game\\">Start New Game</a><a class=\\"block w-full bg-light text-dark\\" href=\\"/continue\\">Continue Game</a></main>"`;
